Game/shaders/fragment_deferred_geometry.glsl

```diff
@@ -18,10 +18,28 @@ layout (location = 4) out vec4 g_emissive;
 in VertexData fragment;
 in flat uint instance;
 
+layout (binding = 25) uniform sampler2D noise;
+uniform float noise_progress;
+
 void main()
 {
     Material material = materialsBuffer.materials[instance];
 
+    // Disolving check
+    bool modify_emisive = false;
+    if (noise_progress != 1.0)
```

Comment:
checking for < instead of != might be better as != for doubles or floats are unsafe.